tx.go

```diff
@@ -282,11 +271,32 @@ func (tx *Tx) Commit() (err error) {
 		}
 	}
 
+	tx.db.commitBuffer.Reset()
```

Comment:
please add this logic to be executed in defer func because whether this commit function success or not. The buffer should be used in other tx, and it should be cleaned up.  For data correctness, please ensure this buffer is cleaned up after the commit function executes.

Reply:
That's a good suggestion😁!

Reply:
I use defer to clean up the commit buffer after allocating the buff.